driver/others/memory.c

```diff
@@ -73,6 +73,16 @@ USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include "common.h"
 
+#ifndef likely
+#ifdef __GNUC__
```

Comment:
How about `#if defined(__GNUC__) || defined(__clang__)` ?

Reply:
is clang likely to understand this (across all relevant versions) ? I copied this from the `blas_server*` files...

Reply:
clang 3.8 does which was released 5 years ago. That's the oldest I have access to. Here's what boost does, https://github.com/boostorg/config/blob/develop/include/boost/config/compiler/clang.hpp#L71-L77